src/ethereum_test_tools/rpc/rpc.py

```diff
@@ -113,3 +119,14 @@ def storage_at_keys(
             storage_value = self.get_storage_at(account, key, block_number)
             results[key] = storage_value
         return results
+
+    def debug_trace_call(self, tr: dict[str, str], block_number: str):
+        """
+        `debug_traceCall`: Returns pre state required for transaction
+        """
+        params = [
+            tr,
+            block_number,
+            {"tracer": "prestateTracer"},
+        ]
+        return self.post_request("debug_traceCall", params)
```

Comment:
Very happy to keep this here, although the `EthRPC` class was to denote then `eth_` for calls. 

Could we move this function to its own class `DebugRPC`? To denote the `debug_` calls?

I'll do the same for the engine calls in the future with `EngineRPC`! 

Maybe for another time though

Reply:
Oh good point.

Reply:
Saving for a later PR, cc @artemd24.